### ISSUE TITLE
[istream.iterator.cons] uses no-longer-defined term "constant initializer"

### ISSUE THREAD

**Issue by timsong-cpp** [cplusplus/draft#4954]
This term was removed by CWG2366.

**Comment by JohelEGP** [comment]
@jwakely's https://wg21.link/LWG3600's resolution is going to introduce another use for it. I suppose that's how you noticed and so he might already be aware of it. I'm calling it out loud just in case.

**Comment by jwakely** [comment]
Yes, and we've already decided that new proposed wording is wrong and needs to change again.

**Comment by frederick-vs-ja** [comment]
Looks like yet another case for LWG2833.

**Comment by jensmaurer** [comment]
@Dani-Hub , do you wish to integrate the undefined term "constant initializer" into LWG2833 ?

**Comment by Dani-Hub** [comment]
Sure, I can do that

**Comment by Dani-Hub** [comment]
I updated now [LWG 2833](https://cplusplus.github.io/LWG/issue2833) by a commenting note.
